examples/00_quick_start/embdotbias_movielens.ipynb

```diff
@@ -408,11 +408,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "data = RecoDataLoader.from_df(train_valid_df,\n",
-    "                                    user_name=USER,\n",
-    "                                    item_name=ITEM,\n",
-    "                                    rating_name=RATING,\n",
-    "                                    valid_pct=0.001)"
+    "data = RecoDataLoader.from_df(\n",
+    "    train_valid_df,\n",
+    "    user_name=USER,\n",
+    "    item_name=ITEM,\n",
+    "    rating_name=RATING,\n",
+    "    valid_pct=0.1\n",
```

Comment:
@jmarrietar there was a bug because the validation percentage was too low and it returned an empty dataset. I made a small change, on this and also some minor format changes.

Can you please review?